library/pcs_cluster.py

```diff
@@ -182,15 +197,21 @@ def run_module():
     # if there is no cluster configuration and cluster should be created do 'pcs cluster setup'
     if state == 'present' and not (cluster_conf_exists or corosync_conf_exists or cib_xml_exists):
         result['changed'] = True
-        # create cluster from node list that was provided to module
+        # create cluster from node list that was provided to module        # create cluster from node list that was provided to module
         if pcs_version == '0.9':
+            # if no transport_options are specified used empty string
+            if (module.params['transport_options']):
+                module.fail_json(msg="using transport_options is not supported with pcs 0.9")
             module.params['token_param'] = '' if (not module.params['token']) else '--token %(token)s' % module.params
             module.params['transport_param'] = '' if (module.params['transport'] == 'default') else '--transport %(transport)s' % module.params
             cmd = 'pcs cluster setup --name %(cluster_name)s %(node_list)s %(token_param)s %(transport_param)s' % module.params
         elif pcs_version == '0.10':
+            # if no transport_options are specified used empty string
```

Comment:
this can be handled by `default=''` for `transport_options` parameter as mentioned earlier which I think looks cleaner